unlock-app/src/components/interface/keychain/QRModal.tsx

```diff
@@ -28,17 +37,36 @@ export const QRModal = ({ active, dismiss, sendEmail, signature }: Props) => {
     console.log(url.toString()) // debugging
     return url.toString()
   }
+  const isEthCC =
+    [
+      '0xd0a031d9f9486b1d914124d0c1fcac2e9e6504fe',
+      '0x072149617e12170696481684598a696e9a4d46ff',
+      '0xf181e18e007517605f369eccf0eee6ebb1b10133',
+      '0x6d8c3d90340fa33693a88d1411b0f32df12d0683',
+      '0xf99eb828ac365c54fcbb6779a78417c25f113829',
+      '0x9ab351cb5dae55abd135dd256726851aae8efeb5',
+      '0x4624bbf6d685b1057eecacc691b0a068e287f0a5',
+      '0x623da3e4d4cb9c98dabb4c23789ed5aaa20ea3aa',
+    ].indexOf(lock.address.toLowerCase()) > -1
+
+  const addToPhone = () => {
+    window.open(
+      `https://ethcc.ethpass.xyz/integrations/ethcc?payload=${encodeURIComponent(
```

Comment:
any explanation for what this is?

Reply:
This is to ad to wallets. It will likely be removed next week.